bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/dto/SurePetcareDeviceCurfew.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.surepetcare.internal.dto;
+
+/**
+ * The {@link SurePetcareDeviceCurfew} class is used to serialize a curfew.
+ *
+ * @author Rene Scherer - Initial contribution
+ */
+public class SurePetcareDeviceCurfew {
+
+    public Boolean enabled;
```

Comment:
does this still apply here? if so what need to be set to boolean?

Reply:
a `Boolean` in Java is an object, so it can be `true`, `false`, and `null` but using it `null` does not seem to make much sense in this context. So by using a boolean you make it explicit that will be either `true` or `false`

Reply:
done